ffwd-client/src/test/java/com/spotify/ffwd/v1/MetricTest.java

```diff
@@ -0,0 +1,119 @@
+/*-
+ * -\-\-
+ * FastForward Java Client
+ * --
+ * Copyright (C) 2016 - 2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.ffwd.v1;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.spotify.ffwd.protocol1.Protocol1;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class MetricTest {
+
+  private static final String HOST = "host";
+  private static final String KEY ="key";
+  private static final String TAG1 = "tag1";
+  private static final String TAG2 = "tag2";
+  private static final String KEYVALUE ="val1";
+  private static final String DISTRIBUTION_DATA_POINT ="FAKEDATAFAKEDATAFAKEDATA";
+
+
+  @Test
+  public void testMetricValueNull_default(){
```

Comment:
i think it would be good to keep everything camelcase like `testMetricValueNullDefault` unless there's a particular reason